src/main/java/org/broadinstitute/hellbender/tools/variantdb/RawArrayData.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.tools.variantdb;
+
+import static org.broadinstitute.hellbender.tools.variantdb.BinaryUtils.*;
+
+public class RawArrayData {
+    public static enum ArrayGenotype {
+        // Order is critical here, the ordinal is the int encoding
+        AA,AB, BB, NO_CALL
+    }
+
```

Comment:
again, i think we should merge this with the enum in the ingest code (i can do this after the PR is merged)